src/main/resources/default/taglib/w/deleteButton.html.pasta

```diff
@@ -1,6 +1,6 @@
 <i:arg type="String" name="url"/>
 <i:arg type="boolean" name="adminOnly" default="false"/>
-<i:arg type="Page" name="page" default=""/>
+<i:arg type="sirius.web.controller.Page" name="page" default=""/>
```

Comment:
not needed

Reply:
Not even when we have a `oxomi.content.catalogs.Page` class in OX?

Reply:
I had clashes with the Page.class in OX so fully qualified the import here.

Reply:
well then we should make the auto import in biz smarter to no create duplicate imports as this will happen again and again

Reply:
done